styles/alert.css

```diff
@@ -0,0 +1,64 @@
+    /*
+        Author: Vrushabh Dhatrak
+        Release: 0.1
+        Description : Alert Component Css
+    */
+    /* Alert Starting Here*/
+    .alert {
```

Comment:
Remove tab in the beginning on all lines. Other CSS files also look similar.

Reply:
Ok Thanx will do